### PR TITLE
Bug1118468 postgres rds

### PR DESCRIPTION
**Pull request by rhelmer**
r? @jdotpz / @phrawzty - not sure about the security groups, the rest is OK I think.

### CODE REVIEW

File: terraform/postgres/main.tf [hunk 1]
```diff
@@ -4,88 +4,29 @@ provider "aws" {
...
+    password = "${var.rds_root_password}"
+    parameter_group_name = "default.postgres9.4"
+    # provisioned IOPS SSD
+    storage_type = "io1"
+    iops = "10000"
+    final_snapshot_identifier = "rds-postgres-${var.environment}-final"
+    publicly_accessible = true
     tags {
-        Name = "${var.environment}__postgres_${count.index}"
+        Name = "rds-postgres-${var.environment}"
```
Comment (jdotpz) [inline terraform/postgres/main.tf]: You should add the tags for:
role (would be postgres)
project (would be socorro)

File: terraform/postgres/main.tf [hunk 2]
```diff
@@ -4,88 +4,29 @@ provider "aws" {
...
-    tags {
-        Environment = "${var.environment}"
-    }
+resource "aws_db_subnet_group" "default" {
+    name = "main"
+    description = "Our main group of subnets"
+    subnet_ids = ["${split(",", var.subnets)}"]
 }
 
-resource "aws_instance" "postgres" {
```
Comment (jdotpz) [inline terraform/postgres/main.tf]: You probably want:
resource "aws_security_group" "rds-socorro-sg" {
    name = "rds-${var.environment}-socorro-sg"
    description = "Socorro RDS security group."
    ingress {
        from_port = 5432
        to_port = 5432
        protocol = "tcp"
-        cidr_blocks = [
-            "172.31.0.0/16"
-        ]
    }
-    egress {
-        from_port = 0
-        to_port = 65535
-        protocol = "tcp"
-        cidr_blocks = [
-            "0.0.0.0/0"
-        ]
-    }
    lifecycle {
        create_before_destroy = true
    }
    tags {
        Environment = "${var.environment}"
        role = "postgres"
        project = "socorro"
    }
}

Then, associate that rds instance like so:
vpc_security_group_ids = "${aws_security_group.rds-socorro-sg.id}"

**Comment by rhelmer** [comment]
@jdotpz better now?

**Comment by jdotpz** [comment]
r+!

**Pull request merged**